content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,382 @@
+
+To learn more about PAS, see the concept Staking, [Penalty-Absorbing Stake (PAS)](/concepts/staking/#penalty-absorbing-stake-pas).
+
+## Rules
+
+### Accountability rules
+
+Accountability rules are applied to detect faults in the three Tendermint consensus round phases *propose*, *prevote*, *precommit*. Rules are detailed in the table below.
+
+{{% alert title="Info" color="info"%}}
+In the table:
+
+- Rule ID: is the unique identifier for the Rule defined in the AFD rule engine. ID prefixes correspond to Tendermint consensus phases:
+  - `P`: *propose*
+  - `PV`: *prevote*
+  - `C`: *precommit*
+{{% /alert %}}
+
+| Rule ID | Description |
+| --| --|
+| `PN` | Proposer has proposed a new block proposal having already sent a _non-nil_ precommit message earlier <!--, or (b) has received sufficiently many prevotes for an earlier proposal block proposal during the same round. --> |
+| `PO` | Proposer has proposed a block proposal that has already been proposed without attaching as justification the `2f + 1` prevotes from the same round for the value. |
+| `PVN` | Committee member has sent two distinct prevotes during the same round. |
+| `PVO` | Committee member has prevoted for a block proposal in more than one consensus round. |
+| `PVO12` | Committee member has prevoted for a block proposal having already precommitted for that block proposal or a different block proposal in an earlier round.  |
+| `PVO3` | Committee member has prevoted for an invalid old block proposal (i.e. sent an invalid message) |
+| `C`  | Committee member has precommitted for an invalid block proposal  (i.e. sent an invalid message) |
+| `C1` | Committee member has precommitted for a block proposal having already precommitted for that block proposal in an earlier round. |
+| `InvalidProposal` | Proposer has proposed a block proposal that fails blockchain validation |
+| `InvalidProposer` | Proposer of a block proposal is not the committee's elected Proposer. |
+| `Equivocation` | Validator has sent conflicting messages during a consensus round: a committee member has sent conflicting votes  (prevote, precommit), a  proposer has broadcast conflicting block proposals to different committee members. |
+| `InvalidRoundStep` | Consensus round message contains invalid round number or step |
+| `AccountableGarbageMessage` | Consensus round message was signed by sender, but it cannot be decoded |
+| `MsgNotFromCommitteeMember` | Consensus round message sender is not a member of the current consensus committee |
+
+### Rule severity
+
+Rules are given a severity rating according to the risk that failure to adhere to the rule brings to block chain finality and integrity. For example:
+
+- failing to finalize a block and/or halting the chain
+- proposing an invalid block
+- voting for multiple and/or conflicting blocks
+
+| Severity | Description |
```

Comment:
As of now we have only `Mid` severity being used

Reply:
That's a good point. I tried that but when I looked at the table it made me think "Where is Low? And why start with Mid?" That seemed strange and confusing as it raised questions.

Then, in the section on slashing protocol configuration (/concepts/accountability/#slashing-protocol-configuration) do you leave out 

>_base slashing rate low_ = `1000` (10%), base slashing rate for a fault of Low severity

**It's in the code and config.**

So I thought it better to put it in, reflect the code in the release, and where appropriate in the table here pub in not currently implemented.  I didn't mark the Low entry as not currently implemented, because there is a config value for it set in the code...

I mean, if you think it is better to remove it and 'less is more', then OK.

But, for consistency it is more than one edit here.  I think necessary editing is:

- removing the Low here in the table
- removing the _base slashing rate low_ config in the configsection  above - /concepts/accountability/#slashing-protocol-configuration
- removing the  _base slashing rate low_ in the config in the genesis Reference - /reference/genesis/#configaccountability-object

---
Note on technical writing approach taken here: 

-  keep the docs to **what is in the code and config** of the AGC as consistently and as simply as possible.

Reply:
>  I didn't mark the Low entry as not currently implemented, because there is a config value for it set in the code...

Then I would keep references to the low severity parameter in the 2 config-related sections you cited (/concepts/accountability/#slashing-protocol-configuration and /reference/genesis/#configaccountability-object) since as you said they are present in the config. However I would comment out this table for now and just write that all offences are considered `Mid` severity for now, since that's what really happens in the code.

That does not introduce inconsistencies, the low parameter is in the config however it is not utilized.

Reply:
Makes sense. I have commented out the table and will add a placeholder note for now:

{{% alert title="Note" %}}
In this release all offences are treated as `Mid` severity.

Future releases will introduce an offence scale range. 
{{% /alert %}}

Reply:
Commit 2401d40bc56baeb2145b5857c022b9a41d88eefb